packages/cli/README.md

```
@@ -1,206 +1,285 @@
+# 5. In another terminal, test it
+curl -X POST http://localhost:3000/collect \
+  -H "Content-Type: application/json" \
+  -d '{"name":"page view","data":{"title":"Home"}}'
+```
+
+## Architecture
 
-logger.info('Starting process...');
-logger.success('Process completed!');
-logger.error('Something went wrong');
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)